docs/configuration.md

```diff
@@ -1043,6 +1043,30 @@ module.exports = {
 };
 ```
 
+### Custom Sequelize Configuration
+
+```js
+const fs = require('fs');
+
```

Comment:
```suggestion
```

this example doesn't use `fs`

Reply:
That's what I get for copy/paste... Good catch.